Packs/CommonScripts/ReleaseNotes/1_10_41.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: Dig
+- DNS lookup utility to provide 'A' and 'PTR' record  (Available from Cortex XSOAR 6.0.0).
```

Comment:
Add here also the release note for the docker image update

Reply:
Fixed.